decidim-core/lib/decidim/translatable_attributes.rb

```diff
@@ -33,8 +33,7 @@ def translatable_attribute(name, type, *options)
 
           define_method attribute_name do
             field = public_send(name) || {}
-            corrected_locale = locale.to_s.gsub("__", "-")
```

Comment:
Mm, the removed `gsub` was introduced in #93 and it doesn't seem that the cause of the problem has disappeared. :thinking:
Also, [another part of that PR's code](https://github.com/decidim/decidim/blob/master/decidim-core/lib/decidim/translatable_attributes.rb#L31) would be kept and it seems to be related with this.

Reply:
I don't think that one is needed, that's why I removed only that one but kept the other. If I'm reading the code properly, the `locale` variable contains a locale (`ca`, `pt-BR` or whatever), not an "locale accessor" (`description_ca`, `description_pt__BR`, or whatever). So the substitution is unnecessary since locales do not contain double dashes.

I might be missing something, but if I'm missing something, we're also missing a spec because this is green :smiley:.

Reply:
Yep, I think that you are right, I didn't notice that this line wasn't doing anything before this PR. :+1: